packages/engine/core/src/lib/connector.ts

```diff
@@ -1,5 +1,7 @@
 import type { ExternalProfile, Profile, Message } from '@remixproject/plugin-utils'
 import { Plugin, PluginOptions } from './abstract'
+import * as crypto from 'crypto'
```

Comment:
Everything in the engine-core was designed to be platform agnostic (browser / node). By adding a dependency on cypto (which is node specific) you might break the engine for the web.

Reply:
what about using https://github.com/ethereumjs/ethereumjs-wallet/blob/master/docs/classes/wallet.md ?